src/components/StageSelect.js

```diff
@@ -11,9 +12,39 @@ const animatedComponents = makeAnimated();
 export class StageSelect extends React.Component {
   constructor(props) {
     super(props);
+    this.state ={
+      flag : false
+    }
+  }
+
+  ConnectMerakiApi = () => {
+    axios.post(`https://connect.merakilearn.org/api/offerLetter/admissions`,{
+      "receiverEmail": "poonam@navgurukul.org",
```

Comment:
@Poonam-Singh-Bagh Why is the receiver your email? It should be student's email.

Reply:
Because you told me to keep it like this. You told me that you will do that one. What you told that only I did.